### DISCUSSION TITLE
Does PVD1 controller is already included in the ESD1 model

### DISCUSSION THREAD

**Discussion by SanjeevKumarGoyal** [CURENT/andes#316] [Q&A]
Good Morning Dr. Cui,

Please help us on below item:

We are using `ieee14_esd1` model for `ESD1` (Battery), and `ieee14_dgprct1` model for `PVD1`.
We want to use `ESD1` (battery) model to perform QV and PV control as well.

Do we need to use `PVD1` as a controller along with `ESD1` to achieve  the PV and QV control or PVD1 controller is already included in the ESD1 model?

Regards,
Sanjeev

**Comment by cuihantao** [comment]
> Do we need to use PVD1 as a controller along with ESD1 to achieve the PV and QV control or PVD1 controller is already included in the ESD1 model?

PVD1 and ESD1 are separate. All in PVD1 are inherited by ESD1, meaning that everything PVD1 has is already in ESD1.
